tests/run_tests.py

```diff
@@ -53,13 +53,8 @@ def RunTest(self, tc):
     cmd += [tc.GetInputPath()]
 
     try:
-      err = subprocess.check_output(cmd, stderr=subprocess.STDOUT)
-      if err != "" and not tc.IsExpectedFail():
-        sys.stdout.write(err)
-        return False
-
+      subprocess.check_output(cmd)
```

Comment:
I thought we decided to leave this as is and make the summary print a command line option?